### PR TITLE
Added to metadataProtocols in config.groovy

### PR DESCRIPTION
**Pull request by pmbohm**
@atkinsn discovered an additional protocol is being used by metadata

### CODE REVIEW

**Comment by dnahodil** [comment]
@atkinsn could you please give an example of a data collection which uses this protocol?

**Comment by atkinsn** [comment]
Uuid: 8af21108-c535-43bf-8dab-c1f45a26088c

File: grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -263,7 +263,8 @@ portal {
 
         supplementary = [
             "WWW:LINK-1.0-http--link",
-            "WWW:LINK-1.0-http--downloaddata"
+            "WWW:LINK-1.0-http--downloaddata",
+            "WWW:LINK-1.0-http—related"
```
Comment (dnahodil) [inline grails-app/conf/Config.groovy]: This looks like it might be the wrong hyphen/dash character @pmbohm?

**Comment by dnahodil** [comment]
Thanks, Nat

File: grails-app/conf/Config.groovy (hunk 1 above)
Comment (dnahodil) [inline grails-app/conf/Config.groovy]: Yeah, this seems to be incorrect. Perhaps a text editor somewhere along the way turned a double hyphen into a dash of some kind.

**Comment by pmbohm** [comment]
@dnahodil additional additional protocol now added

**Pull request merged**